assets/src/settings-page/index.js

```diff
@@ -127,24 +127,29 @@ function Root() {
 	const { downloadingTheme } = useContext( ReaderThemes );
 	const [ saved, setSaved ] = useState( false );
 
+	// Show the success notice after options have saved.
+	useEffect( () => {
+		if ( didSaveOptions ) {
+			setSaved( true );
+		}
```

Comment:
To me it would seem preferable if the notice were hidden immediately upon entering a dirty state, before the 9 seconds are up.

```suggestion
		setSaved( didSaveOptions );
```

Reply:
Handled in e3406e3. See https://github.com/ampproject/amp-wp/pull/5265#discussion_r476755180